src/style.css

```diff
@@ -1,3 +1,100 @@
-body {
+* {
+  font-family: Roboto, sans-serif;
   margin: 0;
-}
+  box-sizing: border-box;
+  }
+
+  body {
+  margin: 0;
+  padding: 0;
+  font-size: 12px;
+  }
+
+  .header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 100%;
+  height: 100%;
+  margin-top: 0;
+  }
+
+  .nav {
+    padding: 0;
+    margin: 0;
+  }
+
+  .nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  font-size: 12px;
+  text-align: center;
+  }
+
+  .nav__item {
+  margin-left: 20px;
+  font-size: 12px;
+  text-align:right;
+  }
+
+  .nav__item:first-child {
+    margin-left: 0;
+  }
+
+  .nav__link {
+    display: block;
+    position: relative;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  text-transform:uppercase;
+  font-style: normal;
+  line-height: 60px;
+  font-weight: 500;
+  text-align: right;
+  padding: 0;
+  margin: 0;
+
+  }
+
+  .nav__link:hover {
+  color: #00ACDC;
+  border-radius: 8px;
+  height: 4px;
+  max-height: 60px;
+  }
+
+  .logo_link {
+    display: block;
+    height: 40px;
+  }
+
+  .logo {
+  width: 40px;
+  height: 40px;
+  }
+
+  .is-active {
+    color: #00ACDC;
+    border-radius: 8px;
+    height: 4px;
+    max-height: 60px;
+    bottom: 0;
+    }
+
+  .is-active::after {
+  position: absolute;
+  bottom: 0;
+  content: "";
+  display: block;
+  background-color: #00ACDC;
```

Comment:
move this color to some var and reuse everywhere

Reply:
corrected